addons/web/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    def _get_session_info(self, allowed_company_ids):
```

Comment:
Can you please rename `allowed_company_ids` to `allowed_companies`?
Also, can you add some docstring to explain what this parameter is for?